.golangci.yml

```diff
@@ -19,3 +19,9 @@ linters:
     - gosimple
     - unconvert
     - gocyclo
+    - gosec
+issues:
+  exclude-rules:
+    - linters:
+      - gosec
+      text: "G110|G601|G404|G204|G306"
```

Comment:
Nit: space at the end of the file 
Also, would be nice add a comment and describe what and why has been excluded.

Reply:
@camilamacedo86 it would be a lot to add all these in on one PR.  In particular there has already been some debate on the solution for some.  This PR was intended to be the easy previously agreed upon solutions.
I'm happy to help resolve all of these... can we separate out that work?
I'm also happy to annotate the code location of the lint issue instead of globally... I went global here because without the linter (which was the previous state of things), it was a global ignore in the first place.   I expect G601: memory aliasing to be many places of code and will come with some debate (at least it did previously).
By having this global here now, we are indicating that we are "slowly" adding in this linter and provides a plan of attack for someone new to the project.  It is also consistent with kubernetes proper which started adding in linters and has a file listing sections of kubernetes and category of lint that is failing.